composer.json

```diff
@@ -25,6 +25,7 @@
         "ext-pdo": "*",
         "ext-redis": "*",
         "ext-xml": "^7.2",
+        "ext-yaml": "*",
```

Comment:
This need to be inside "suggest" (https://getcomposer.org/doc/04-schema.md#suggest). So you can install it only when you really need.  And in case if you try to use it with `ext-yaml` - throw expection that it need to be installed.

Reply:
I added it to `require-dev` also, because it's requried for the tests